src/pudl/transform/eia861.py

```diff
@@ -2440,6 +2487,7 @@ def core_utility_data_eia861(raw_eia861__utility_data: pd.DataFrame):
         "core_eia861__yearly_operational_data_revenue": AssetIn(),
         "core_eia861__yearly_reliability": AssetIn(),
         "core_eia861__yearly_sales": AssetIn(),
+        "core_eia861__yearly_short_form": AssetIn(),
```

Comment:
This should definitely be here! But also, I could see it having some downstream consequences on the service territory tables, since I imagine there are utilities we've never captured before, since they're small and have only ever report the short form.

Has the full ETL and data validations been run with this change to catch any issues that might arise?

Also, I suspect that the new table should also be included as an input to the asset below: `core_eia861__assn_balancing_authority`

Reply:
I tried running the full ETL and the docs build but it kept showing an error about 'incorrect Google cloud credentials'. 

Recently, I cleaned my Linux and tried building the PUDL project but unfortunately, I'm still getting an error involving Google cloud. This time the error is on corrupt packages and these packages all come from the google-cloud-sdk folder. 

Please do you know why this is happening? I've not been able to move forward because of this.

Reply:
How are you running the full ETL? At what stage are you getting the Google cloud credential errors? It should definitely be possible to run the ETL without any credentials. Do you have all of the raw input data downloaded locally already? Is it maybe trying to re-download that data from the cache of Zenodo data that we have up on GCS?

Have you tried removing the `pudl-dev` conda environment and rebuilding it?

Reply:
> How are you running the full ETL? At what stage are you getting the Google cloud credential errors? It should definitely be possible to run the ETL without any credentials. Do you have all of the raw input data downloaded locally already? Is it maybe trying to re-download that data from the cache of Zenodo data that we have up on GCS?
> 
> Have you tried removing the `pudl-dev` conda environment and rebuilding it?

Hello Zane, 
Sorry for the late reply. Yes, I successfully rebuilt it today. I think the problem was my old computer because building it on my new computer went smoothly. I'll work on your comments tomorrow and run the tests before pushing them. Hopefully, I don't get an error. I'll keep you updated.